terraform/README.md

```diff
@@ -0,0 +1,129 @@
+# Deploying Registry-Sweepers ECS Tasks
+
+## Setting Up SSH Authentication for a Private Terraform Module Repository
+Since **pds-tf-module** is a private repo, pulling the ECS module code will require SSH Access. To access a private Terraform module repository, you need to generate an SSH key and add it to your GitHub/GitLab account.
+
+**Note:** If you already have an existing SSH key for GitHub access, you can simply.
```

Comment:
You can simply … ? 

Incomplete sentence, but not a deal-breaker, continuing to review